src/math/floor.js

```diff
@@ -0,0 +1,24 @@
+const floor = (number, precision) => {
+    let numberString = number.toString();
+    let decimalIndex = numberString.indexOf('.');
+    let beforeDecimal = numberString.substring(0, decimalIndex);
+    let afterDecimal = numberString.substring(decimalIndex, numberString.length);
+  if (Number.isInteger(number)){
+    return number;
+  } else if (Number.isInteger(number) === false) {
```

Comment:
If something isn't true we can assume that it is false. It's a binary set of results.